rules/okta_rules/okta_threatinsight_security_threat_detected.py

```diff
@@ -0,0 +1,42 @@
+from panther_base_helpers import okta_alert_context
+
+
+def severity_from_threat_string(threat_detection):
+    # threat detection is a string but contains json data
+    # can contain multiple threats detected with multiple severities
+    # return highest found severity
+    if "CRITICAL" in threat_detection:
+        return "CRITICAL"
+    if "HIGH" in threat_detection:
+        return "HIGH"
+    if "MEDIUM" in threat_detection:
+        return "MEDIUM"
+    if "LOW" in threat_detection:
+        return "LOW"
+    if "INFO" in threat_detection:
+        return "INFO"
```

Comment:
```suggestion
        return "LOW"
```

In reviewing the example tests/logs below, I suggest we don't go below `LOW`

Reply:
@calkim-panther - what do you think?

Reply:
(if okta found a threat, and we mark as INFO, a customer will never see it)

Reply:
I'm leaning toward preserving it. We would only pass it as INFO if that is the highest detection Okta passes in the alert. In reality, i don't think they would ever send a threatinsight alert with just an INFO detection, but in case they do I don't feel strongly about raising it to LOW or not